### DISCUSSION TITLE
Categories are not showing in the Navigation menu in Header

### DISCUSSION THREAD

**Discussion by naveenraj08** [breezefront/community#181] [5. General]
Hi Mate!
I am new to the Breeze theme so, I don't know how to display the categories in the Navigation menu, but in Luma by default, the categories will come in the Navigation menu, but in the Breeze theme I can't able to find the answer for that, so can anyone explain or make me understand how to display the categories in Navigation menu.

**Comment by vovayatsyuk** [comment]
You don't have to do anything to show categories. They should appear automatically just like in the Luma theme.

I don't know why it's not working for you. Try to disable all third-party modules you have.
